content/blog/2020/2020-09-02-rackspace-best-practise-to-clone-a-peoplesoft-environment/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Rackspace best practise to clone a Peoplesoft environment"
```

Comment:
I'm seeing both "Peoplesoft" and "PeopleSoft" throughout, so we should probably be consistent there